modules/node-shared/src/main/scala/org/tessellation/node/shared/cli/CliMethod.scala

```diff
@@ -61,6 +61,8 @@ object CliMethod {
 
   val forkInfoStorageConfig: ForkInfoStorageConfig = ForkInfoStorageConfig(10)
 
+  val doubleSignDetectConfig: DoubleSignDetectConfig = DoubleSignDetectConfig(10, 1.minute)
```

Comment:
it would be great to explicitly name arguments (as you can see above in the file)

Reply:
Added names to the arguments.